pkg/templates/aws/resources/load_balancer.yaml

```diff
@@ -91,4 +99,4 @@ classification:
 delete_context:
   requires_no_downstream: true
 views:
-  dataflow: big
```

Comment:
i know we were discussing this, are you just going with the approach on the ui to flip this for nlb?